FRC1076-2016-Competition/src/org/usfirst/frc/team1076/robot/physical/Robot.java

```diff
@@ -127,8 +127,32 @@ public void robotInit() {
 		armFollower.enableBrakeMode(true);
 		armExtendMotor.enableBrakeMode(true);
 		armExtendFollower.enableBrakeMode(true);
+		
+		armMotor.ConfigFwdLimitSwitchNormallyOpen(true);
+		armMotor.ConfigRevLimitSwitchNormallyOpen(true);
+		armMotor.enableLimitSwitch(true, true);
+		armExtendMotor.ConfigFwdLimitSwitchNormallyOpen(true);
+		armExtendMotor.ConfigRevLimitSwitchNormallyOpen(true);
+		armExtendMotor.enableLimitSwitch(true, true);
 		// leftFollower.changeControlMode(TalonControlMode.Follower);
 		// leftFollower.set(LEFT_INDEX);
+
+		// These are the physical limit switches. By default it
+		// initalizes in the "normally open" state.
+		armMotor.enableLimitSwitch(true, true);
+		armFollower.enableLimitSwitch(true, true);
+		// Soft Limits are limits which disable the motor if
+		// the "Sensor Position" (the encoder) is outside a certain range
+		// This is disabled in teleop.
+		// TODO: Find true soft limits
+		armMotor.setForwardSoftLimit(10000);
+		armMotor.setReverseSoftLimit(10000);
```

Comment:
Since we're manually controlling both motors, don't the followers need to have the same limits set on them?